src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -116,6 +122,17 @@ export const TradeConfirm = ({ history }: RouterProps) => {
   const isFeeRatioOverThreshold =
     gasFeeToTradeRatioPercentage > gasFeeToTradeRatioPercentageThreshold
 
+  const txLink = useMemo(() => {
+    if (
+      trade.sellAsset.chainId === KnownChainIds.OsmosisMainnet ||
+      trade.sellAsset.chainId === KnownChainIds.CosmosMainnet
```

Comment:
```suggestion
fromChainId(trade.sellAsset).chainNamespace === CHAIN_NAMESPACE.COSMOS
```

This will be more future-proof, it will support all IBC chains automagically since they all share the Cosmos chain namespace

Reply:
bug in your suggested change.  should be `trade.sellAsset.chainId`

Reply:
Also CHAIN_NAMESPACE.Cosmos